src/cljs/nr/gameboard/board.cljs

```diff
@@ -1554,13 +1554,23 @@
           [:span link " " [:span {:class "anr-icon link"}] (str " + " )]
           (let [strength (if bonus (+ base bonus) base)]
             [:span (str strength " + ")]))))
-    [:select#credit {:onKeyUp #(when (= "Enter" (.-key %))
+    [:select#credit {:value (or (get-in @app-state [:select :credits]) 0)
```

Comment:
This will cause a re-render when `app-state` changes. If you need a local bit of state, you can use a form-2 or `r/let` over an ratom.